packages/next/src/client/components/navigation.ts

```diff
@@ -33,6 +33,7 @@ export class ReadonlyURLSearchParams {
   keys: URLSearchParams['keys']
   values: URLSearchParams['values']
   toString: URLSearchParams['toString']
+  size: URLSearchParams['size']
```

Comment:
Maybe temporarily use `any` for size if that's not part of URLSearchParams yet. Or update the global type overrides we use to add it?
```suggestion
  size: any | URLSearchParams['size']
```